### PR TITLE
Issue #375 remove path model

### LINKED ISSUE #375: Remove path model
Considering that 
  * we have removed the single language restriction per guide
  * we have introduced tenants
  * we are decided to order categories to create a single path per tenant

it doesn't have any sense to have a path model. It should be merged into category, and category renamed to something like `chapter`

### PR DESCRIPTION
**Pull request by flbulgarelli**
fixes #375

@faloi  @fedescarpa

### CODE REVIEW

File: app/admin/path_rule.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-ActiveAdmin.register PathRule do
+ActiveAdmin.register ChapterGuide do
```
Comment (faloi) [inline app/admin/path_rule.rb]: The file name should be `chapter_guide`

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -39,7 +39,7 @@ def tab_list(tabs)
   end
 
   def path_finished(guide)
```
Comment (faloi) [inline app/helpers/application_helper.rb]: Shouldn't it be `chapter_finished`?

File: config/locales/en.yml [hunk 3]
```diff
@@ -138,7 +137,7 @@ en:
   need_a_hint: "Do you need a hint?"
   about: "About"
   sending_solution: "Sending solution"
-  path_finished_html: "You have finished %{path}!"
+  path_finished_html: "You have finished %{chapter}!"
```
Comment (faloi) [inline config/locales/en.yml]: It should be `guide_finished_html`

Comment (faloi) [inline config/locales/en.yml]: *chapter, not guide

File: config/locales/es.yml [hunk 4]
```diff
@@ -166,7 +165,7 @@ es:
   recent_activity: "Actividad Reciente"
   created_exercises: "Ejercicios Creados"
   created_guides: "Guías Creadas"
-  path_finished_html: "¡Terminaste %{path}! ¡Felicitaciones!"
+  path_finished_html: "¡Terminaste %{chapter}! ¡Felicitaciones!"
```
Comment (faloi) [inline config/locales/es.yml]: Idem en.yml

File: spec/models/chapter_spec.rb [hunk 5]
```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
+
+describe Chapter do
+  let!(:functional) { build(:chapter, name: 'Functional Programming') }
+  let!(:haskell) { build(:language, name: 'haskell') }
+  let(:guide) { build(:guide, github_repository: 'flbulgarelli/mumuki-sample-exercises', author: 'author', chapter: functional) }
+
+  it { expect(functional.name).to eq 'Functional Programming' }
```
Comment (faloi) [inline spec/models/chapter_spec.rb]: Why does this test matter?

File: spec/spec_helper.rb [hunk 6]
```diff
@@ -34,3 +34,9 @@
 
 CodeClimate::TestReporter.start
 
+
+RSpec::Matchers.define :json_eq do |expected_json_hash|
```
Comment (faloi) [inline spec/spec_helper.rb]: Great piece of code! I wonder who was the author...

File: app/views/chapters/index.html.erb [hunk 7]
```diff
@@ -0,0 +1,25 @@
+<h2 class="text-center"><%= t :categories_index_title %></h2>
```
Comment (aguspina) [inline app/views/chapters/index.html.erb]: This one shouldn't be `chapters_title` ?

File: config/locales/en.yml (hunk 3 above)
Comment (aguspina) [inline config/locales/en.yml]: @faloi You can edit your comments, I'll teach if you want

**Pull request merged**